assets/common/src/scripts/validate_model.py

```diff
@@ -91,12 +93,15 @@ def _load_and_prepare_data(test_data_path: Path, mlmodel: Dict, col_rename_map:
     return data
 
 
-def _load_and_infer_model(model_dir, data):
+def _load_and_infer_model(model_dir, data, task_name):
     if data is None:
         print("Data not shared. Could not infer the loaded model")
         return
 
     try:
+        if task_name == SUMMARIZATION:
+            os.environ["MLFLOW_HUGGINGFACE_USE_DEVICE_MAP"] = "False"
```

Comment:
this is specific to oss transformers? Please add a check to distinguish b/w hftransformers and oss transformers?